tests/nodeos_chainbase_allocation_test.py

```diff
@@ -79,6 +79,9 @@ def isSetProdsBlockNumIrr():
 
     producerNode.kill(signal.SIGTERM)
 
+    # wait for all blocks to arrive and be processed by irrNode
+    time.sleep(1)
```

Comment:
If for simplicity we don't verify all blocks have arrived, we might as well wait a little longer like 3 seconds to prevent flakeness.